### PR TITLE
feat(cancellation): Added cancellation functionality to mail pieces

### PR DESCRIPTION
**Pull request by shrav**

### CODE REVIEW

**Comment by shrav** [comment]
Ready for review @robinjoseph08

File: spec/lob/v1/check_spec.rb [hunk 1]
```diff
@@ -71,4 +72,25 @@
...
+
+      new_check = subject.checks.create(
+        bank_account: new_bank_account["id"],
+        to: new_address["id"],
+        from: new_address["id"],
+        amount: 2000
+      )
+
+      result = subject.checks.destroy(new_check["id"])
+      result["id"].must_equal(new_check["id"])
```
Comment (robinjoseph08) [inline spec/lob/v1/check_spec.rb]: Can you check the `deleted` property as well for all these tests?

File: spec/lob/v1/postcard_spec.rb [hunk 2]
```diff
@@ -114,4 +114,21 @@
...
+      new_address = subject.addresses.create @sample_address_params
+
+      new_postcard = subject.postcards.create(
+        description: @sample_postcard_params[:description],
+        to: new_address["id"],
+        front: File.new(File.expand_path("../../../samples/postcardfront.pdf", __FILE__)),
+        back: File.new(File.expand_path("../../../samples/postcardback.pdf", __FILE__))
+      )
+
+      result  = subject.postcards.destroy(new_postcard["id"])
```
Comment (robinjoseph08) [inline spec/lob/v1/postcard_spec.rb]: nit: extra space before the equal sign

**Comment by robinjoseph08** [comment]
LGTM

File: spec/lob/v1/check_spec.rb (hunk 1 above)
Comment (shrav) [inline spec/lob/v1/check_spec.rb]: Done

**Comment by mgartner** [comment]
LGTM

**Pull request merged**